_pages/how-to-submit.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: splash
```

Comment:
@kierisi i lied - one more comment outside of a content review - you mentioned the TOC... can you try one more thing. i suspect the TOC isn't available for the splash page layout. this is because that layout is a flowing layout not intended to have indexable sections. Try to make this a "single" layout page and see what happens. you can always use the wide class like we do with blogs.

Reply:
@lwasser that did it! I commit those changes and it should be formatted now.

Reply:
perfect!! yay!